manifests/g/GeoGebra/Classic/6.0.811/GeoGebra.Classic.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Automatically updated by the winget bot at 2023/Nov/11
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: GeoGebra.Classic
+PackageVersion: 6.0.811
+PackageLocale: en-US
+Publisher: International GeoGebra Institute
+PublisherUrl: https://www.geogebra.org
+PublisherSupportUrl: https://help.geogebra.org/topic/help-support

```

Comment:
```suggestion
# PublisherSupportUrl: https://help.geogebra.org/topic/help-support
```

Reply:
@stephengillie - are you able to commit suggestions on open PRs?

Reply:
Any clue how to make that happen? I thought it would already work :(

Reply:
> Any clue how to make that happen? I thought it would already work :(

I think that the API call the bot makes needs the parameter `maintainer_can_modify=true`
https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#create-a-pull-request

Reply:
I'll check with the team to see who can make that change.